src/Data/Fin/Properties.agda

```diff
@@ -456,6 +456,10 @@ inject₁ℕ≤ = ℕₚ.<⇒≤ ∘ inject₁ℕ<
 ℕ<⇒inject₁< : ∀ {i : Fin (ℕ.suc n)} {j : Fin n} → j < i → inject₁ j < i
 ℕ<⇒inject₁< {i = suc i} (s≤s j≤i) = ≤̄⇒inject₁< j≤i
 
+≤-inj→suc : i ≤ inject₁ j → i ≤ suc j
```

Comment:
Could we name this `i≤inject₁[j]⇒i≤1+j` please?

Reply:
I renamed it.